src/levelEditor/preview/LevelPreviewEntities.js

```diff
@@ -53,11 +54,38 @@ function LevelPreviewEntities(props: Props): React$Node {
 		}
 
 		return levelDecos.map((dec, index) => {
+			const urlPrefix = process.env.REACT_APP_SPRITES_URL_PREFIX;
+			const originx = spritesData[dec.spr].originx;
+			const originy = spritesData[dec.spr].originy;
+
+			let decoElement;
+			if (urlPrefix == null) {
+				// Add sprite toggle here
+				decoElement = <>{dec.spr}</>;
```

Comment:
React fragment not needed here
```suggestion
				decoElement = dec.spr;
```